src/nvim/version.c

```diff
@@ -43,54 +43,17 @@ char *version_cflags = "Compilation: " NVIM_VERSION_CFLAGS;
 #ifdef INCLUDE_GENERATED_DECLARATIONS
 # include "version.c.generated.h"
 #endif
-static char *(features[]) = {
+static char *features[] = {
 #ifdef HAVE_ACL
   "+acl",
 #else  // ifdef HAVE_ACL
   "-acl",
 #endif  // ifdef HAVE_ACL
-  "+arabic",
-  "+autocmd",
-  "-browse",
-#ifdef NO_BUILTIN_TCAPS
-  "-builtin_terms",
-#endif  // ifdef NO_BUILTIN_TCAPS
-#ifdef SOME_BUILTIN_TCAPS
-  "+builtin_terms",
-#endif  // ifdef SOME_BUILTIN_TCAPS
-#ifdef ALL_BUILTIN_TCAPS
-  "++builtin_terms",
-#endif  // ifdef ALL_BUILTIN_TCAPS
-  "+byte_offset",
-  "+cindent",
-  "-clipboard",
-  "+cmdline_compl",
-  "+cmdline_hist",
-  "+cmdline_info",
-  "+comments",
-  "+conceal",
-  "+cscope",
-  "+cursorbind",
-  "+cursorshape",
-  "+dialog_con",
-  "+digraphs",
-  "-dnd",
-  "-ebcdic",
-  "-emacs_tags",
-  "+eval",
-  "+ex_extra",
-  "+extra_search",
-  "+farsi",
-  "+file_in_path",
-  "+find_in_path",
-  "+folding",
-  "-footer",
 
   // only interesting on Unix systems
 #if defined(UNIX)
   "+fork()",
```

Comment:
I think it's be redundant, as it looks like `f_system` was reimplemented on top of libuv, and how libuv does things is generally out of the scope of our docs. I'll just remove it.